error.go

```diff
@@ -18,68 +18,69 @@ package edgedb
 
 import (
 	"errors"
-	"fmt"
-	"log"
 
 	"github.com/edgedb/edgedb-go/internal/buff"
 )
 
-var (
-	// Error is wrapped by all edgedb errors.
-	Error error = errors.New("")
+var errZeroResults error = &noDataError{msg: "zero results"}
 
-	// ErrReleasedTwice is returned if a PoolConn is released more than once.
-	ErrReleasedTwice = fmt.Errorf(
-		"connection released more than once%w", Error,
-	)
+// Error is wrapped by all errors.
+type Error interface {
+	Error() string
+	isError()
```

Comment:
Why is `isError()` needed?

Reply:
see https://github.com/edgedb/edgedb-go/pull/51#discussion_r553588923